docs/how-to-guides/azure_resource_provision.json

```diff
@@ -13,7 +13,7 @@
         "principalId": {
             "type": "String",
             "metadata": {
-                "description": "Specifies the principal ID assigned to the role. You can find it by logging into 'https://shell.azure.com/bash' and run 'az ad signed-in-user show --query objectId -o tsv'"
```

Comment:
Create a separate PR to fix all occurrence of this. Please take a look.
https://github.com/linkedin/feathr/pull/360

Reply:
We always want to provision the webapp, so its not an optional feature since that would mean we are not provisioning UI or API server